Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -1819,4 +1819,84 @@ Import aborted.</value>
     <value>The component '{0}' would have been affected by the refactoring, but its state in Rubberduck's memory is stale. Please refresh Rubberduck and try again.</value>
     <comment>{0} stale module</comment>
   </data>
+  <data name="ImplementInterface_TODO" xml:space="preserve">
+    <value>'TODO implement interface member</value>
```

Comment:
This was recently moved to `Refactorings.resx`.  Looks like it is trying to sneak back into the `RubberduckUI.resx`.  :)

Reply:
I'll remove it. That also explains why there was a merge conflict at all.